qp_shogun/sortmerna/utils.py

```diff
@@ -0,0 +1,177 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
```

Comment:
This is a copy from other file, right? If that's correct, could you simply import from that other package vs. copy the full file?

Reply:
The one under sortmerna has a slightly modified version to account for the uncompressed output files from the Sortmerna command.